integration-cli/docker_cli_save_load_unix_test.go

```diff
@@ -86,3 +86,21 @@ func (s *DockerSuite) TestSaveAndLoadWithProgressBar(c *check.C) {
 	expected := fmt.Sprintf("The image %s:latest already exists, renaming the old one with ID", name)
 	c.Assert(out, checker.Contains, expected)
 }
+
+// fail because load didn't receive data from stdin
+func (s *DockerSuite) TestLoadNoStdinFail(c *check.C) {
+	pty, tty, err := pty.Open()
+	c.Assert(err, check.IsNil)
+	cmd := exec.Command(dockerBinary, "load")
+	cmd.Stdin = tty
+	cmd.Stdout = tty
+	cmd.Stderr = tty
+	c.Assert(cmd.Start(), check.IsNil)
```

Comment:
Well, `Start() && Wait()` is the same thing as `Run()`.
What we can do is use https://golang.org/pkg/os/exec/#CommandContext and set a timeout.